rest_framework/fields.py

```diff
@@ -612,6 +612,18 @@ def __init__(self, regex, **kwargs):
         validator = RegexValidator(regex, message=self.error_messages['invalid'])
         self.validators.append(validator)
 
+    def __deepcopy__(self, memo):
+        # Handling case when regex is not passed in kwargs.
+        args = self._args[:1] + copy.deepcopy(self._args[1:])
+        kwargs = dict(self._kwargs)
+
+        # deepcopy doesn't work on compiled regex pattern
```

Comment:
Wondering if it'd instead be simpler to handle all this logic alongside the similar 'validators' special casing in `Field.__deepcopy__`.